src/equisolve/numpy/models/linear_model.py

```diff
@@ -32,6 +36,20 @@ class Ridge:
     :param parameter_keys:
         Parameters to perform the regression for. Examples are ``"values"``,
         ``"positions"``, ``"cell"`` or a combination of these.
+
+    :param solver:
```

Comment:
The solver seems not to be an paramater of the Ridge class but of the fit function. I would out the parameter here as you done it for the doc string.